spec/fields/access_condition_spec.rb

```diff
@@ -16,6 +16,7 @@ def mods_display_access_condition(mods_record)
     @custom_use_reproduction = Stanford::Mods::Record.new.from_str(custom_use_reproduction_fixture).accessCondition
     @copyright_note = Stanford::Mods::Record.new.from_str(copyright_access_fixture).accessCondition
     @cc_license_note = Stanford::Mods::Record.new.from_str(cc_license_fixture).accessCondition
+    @legacy_cc_license_note = Stanford::Mods::Record.new.from_str(legacy_cc_license_fixture).accessCondition
```

Comment:
Can this be moved into the describe block that uses it?  It's hard to hunt though everything up here and there's no reason to initialize it for the other blocks.

Reply:
Agree. Refactored.